app/models/manageiq/providers/amazon/cloud_manager/event_catcher/runner.rb

```diff
@@ -28,7 +28,7 @@ def process_event(event)
   private
 
   def filtered?(event)
-    filtered_events.include?(event["eventType"])
+    @ems.filtered_event_names.include?(event["eventType"])
```

Comment:
I think this can be just 
```suggestion
    filtered_event_names.include?(event["eventType"])
```

because the base class provides the `filtered_event_names` method that goes through the `@ems`

Reply:
Oh no, I misread, the method is on the EMS, but this is the runner.